docs/UserGuide.adoc

```diff
@@ -67,105 +72,112 @@ Examples:
 
 * `help`
 
-* `man add`
+* `man task`
```

Comment:
I don't know if it'll help much but since it's easy lets just include "help task" and "man" in the example? cos if not it may seem like help cannot put command word and man must include command word